devtools/builder/run_docker_build_conda_linux.sh

```diff
@@ -12,11 +12,12 @@ cat << EOF | docker run -i \
                         bash || exit $?
 
 set -x
+set -e
 cd /feedstock_root/
 
 export PATH=\$HOME/miniconda3/bin:\$PATH
 
-for pyver in 3.7 2.7 3.5 3.6; do
+for pyver in 2.7 3.5 3.6 3.7 3.8; do
```

Comment:
I think I will stop to support 2.7 (since ambertools has plan to switch to python 3 now). 

What do you think about dropping 3.5 too?